test/icons/icon.visual-diff.js

```diff
@@ -0,0 +1,41 @@
+const puppeteer = require('puppeteer');
+const VisualDiff = require('visual-diff');
+
+describe('d2l-icon', function() {
+
+	const visualDiff = new VisualDiff('icon', __dirname);
+
+	let browser, page;
+
+	before(async() => {
+		browser = await puppeteer.launch();
+		page = await browser.newPage();
+		await page.setViewport({width: 800, height: 800, deviceScaleFactor: 2});
+		await page.goto(`${visualDiff.getBaseUrl()}/test/icons/icon.visual-diff.html`, {waitUntil: ['networkidle0', 'load']});
+		await page.bringToFront();
+	});
+
+	beforeEach(async() => {
```

Comment:
Looks like you probably don't need this since focus isn't being applied in the tests.  I was using this in the button visual-diff tests to reset the focus between tests without reloading the page.